Gruntfile.js

```diff
@@ -154,15 +154,58 @@ module.exports = function (grunt) {
     preloadSpreadsheet: {
       'v0.2': 'https://docs.google.com/spreadsheets/d/1ughCy983eK-SPIcDYPsjOitVZzY10WdI2MGGrmxzxF4/pubhtml',
     },
-    buildSwf: {
-      dev: {
-        swfPath: "./.tmp/storage.swf",
-        asPath: "./jsflash/dev/Storage.as"
+
+    mxmlc: {
+        options: {
+          // Task-specific options go here.
+        },
+        dev: {
+          files:{
+            './.tmp/storage.swf' : ['./jsflash/dev/Storage.as']
+          }
+        },
+        prod: {
+          files:{
+            './.tmp/storage.swf' : ['./jsflash/prod/Storage.as']
```

Comment:
quick explanation: `grunt serve` compiles to and serves stuff from .tmp. The production distribution is a copy of everything in the dist folder after running `grunt build`, which includes some (but not all) things copied from .tmp.

This setup comes from `generator-angular`, I didn't write it: https://github.com/yeoman/generator-angular